src/core/ext/filters/client_channel/lb_policy/xds/xds.cc

```diff
@@ -273,7 +273,7 @@ class XdsLb : public LoadBalancingPolicy {
   bool PickFromRoundRobinPolicyLocked(bool force_async, PendingPick* pp,
                                       grpc_error** error);
   void UpdateConnectivityStateFromRoundRobinPolicyLocked(
-      grpc_error* rr_state_error);
+      grpc_error* state_error);
```

Comment:
Suggest changing all variables that previous included the string `rr_` to instead use `child_`.

Reply:
Juanli had the same comment, and i could not see a reason why.
With both of you pointing out the same naming change, has convinced me to make the change. So I will.